app/helpers/application_helper.rb

```diff
@@ -35,14 +36,13 @@ def current_locale
     I18n.locale
   end
 
-  def billable_entry_checkbox(entry)
+  def billable_entry_checkbox(entry, entry_type)
     if entry.billed
       "√"
     else
-      tag(:input, type: "checkbox", name: "entries_to_bill[]", class: "bill_checkbox", value: entry.id, "data-project-id" => entry.project.id)
+      tag(:input, type: "checkbox", name: "entries_to_bill[]", class: "bill_checkbox", value: "#{entry_type}-#{entry.id}", "data-project-id" => entry.project.id, "data-entry_type" => entry_type)
```

Comment:
Why do you define information about entry type and entry id twice?
`value: "#{entry_type}-#{entry.id}"`
and
`"data-project-id" => entry.project.id, "data-entry_type" => entry_type`